docs/v6-migration.md

```diff
@@ -4,42 +4,56 @@ title: v6 Migration Guide
 position: 0
 ---
 
-### Updates Since Initial
+### Updates Since Initial Pre-Release
 
--   qBittorrent users should stick with v5.9.2 for the time being if you use Auto Torrent Management in **ANY** capacity.
+Since the initial pre-release (`6.0.0-0`) we've worked with our users and friends to iron out implementations in the most reasonable and usable way possible. As we proceed through the pre-release/testing phase for v6, there may be alterations to things further. They will all be documented here prior to releases.

```

Comment:
```suggestion
Since the initial pre-release (`6.0.0-0`) we've worked with our users and friends to iron out implementations in the most reasonable and usable way possible. As we proceed through the pre-release/testing phase for v6, further changes may be made. They will all be documented here prior to releases.

```